tests/components/ozw/test_websocket_api.py

```diff
@@ -0,0 +1,57 @@
+"""Test OpenZWave Websocket API."""
+
+from homeassistant.components.ozw.websocket_api import ID, NODE_ID, OZW_INSTANCE, TYPE
+
+from .common import setup_ozw
+
+
+async def test_sensor(hass, generic_data, hass_ws_client):
+    """Set up config entry."""
+    await setup_ozw(hass, fixture=generic_data)
+    client = await hass_ws_client(hass)
+
+    """Test network status."""
```

Comment:
Use a comment instead.
```suggestion
    # Test network status.
```